backend/api/user.go

```diff
@@ -118,11 +179,29 @@ func ModifyInfo(c *gin.Context) {
 	c.JSON(http.StatusOK, gin.H{"result": "success"})
 }
 
+// GetUserID godoc
+// @Summary      get my uid
+// @Description  get my uid in the session
+// @Tags         user
+// @Produce      json
+// @Success      200  {object}  model.UIDResponse "success"
```

Comment:
What does this "success" mean?

Does the response message reply success at the reponse body?